frontend/app/src/components/About.vue

```diff
@@ -87,153 +87,171 @@ const versionText = computed(() => {
-                </td>
-              </tr>
-              <tr>
-                <td class="font-medium" :class="css.label">
-                  {{ t('about.frontend_version') }}
-                </td>
-                <td>
-                  {{ frontendVersion }}
-                </td>
-              </tr>
-              <tr v-if="webVersion">
-                <td class="font-medium" :class="css.label">
-                  {{ t('about.platform') }}
-                </td>
-                <td>{{ webVersion.platform }}</td>
-              </tr>
-              <tr v-if="webVersion">
-                <td class="font-medium" :class="css.label">
-                  {{ t('about.user_agent') }}
-                </td>
-                <td>{{ webVersion.userAgent }}</td>
-              </tr>
-              <tr v-if="electronVersion">
-                <td class="font-medium" :class="css.label">
-                  {{ t('about.platform') }}
-                </td>
-                <td>
-                  {{ electronVersion.os }} {{ electronVersion.arch }}
-                  {{ electronVersion.osVersion }}
-                </td>
-              </tr>
-              <tr v-if="electronVersion">
-                <td class="font-medium" :class="css.label">
-                  {{ t('about.electron') }}
-                </td>
-                <td>
-                  {{ electronVersion.electron }}
-                </td>
-              </tr>
-              <tr v-if="componentsVersion">
-                <td colspan="2">
-                  <VDivider class="mt-4 mb-2" />
-                  <div class="font-bold mb-1">
-                    {{ t('about.components.title') }}
-                  </div>
-                </td>
-              </tr>
-              <tr v-if="componentsVersion?.version">
-                <td class="font-medium" :class="css.label">
-                  {{ t('about.components.version') }}
-                </td>
-                <td>{{ componentsVersion.version }}</td>
-              </tr>
-              <tr v-if="componentsVersion?.build">
-                <td class="font-medium" :class="css.label">
-                  {{ t('about.components.build') }}
-                </td>
-                <td>
-                  <DateDisplay :timestamp="componentsVersion.build / 1000" />
-                </td>
-              </tr>
-            </tbody>
-          </table>
-        </VCol>
-        <VCol cols="auto">
-          <CopyButton
-            :value="versionText"
-            :tooltip="t('about.copy_information_tooltip')"
-          />
-        </VCol>
-      </VRow>
-    </VCardText>
-  </VCard>
+                  </template>
+                  <span :class="css.directory">
+                    {{ dataDirectory }}
+                  </span>
+                </RuiTooltip>
+                <div v-if="!isPackaged" class="ml-2">
+                  <RuiTooltip :popper="{ placement: 'top' }" open-delay="400">
+                    <template #activator="{ on, attrs }">
```

Comment:
Are the `on, attrs` needed with `RuiTooltip`?
I have been removing them and everything seems to be fine.